R/tree_accessors.R

```diff
@@ -2904,6 +2904,23 @@ setMethod("col_fnotes_here<-", "LayoutColLeaf", function(obj, value) {
   obj
 })
 
+#' @export
+#' @rdname int_methods
+setMethod(
+  "col_fnotes_here", "VTableTree",
```

Comment:
This extracts all fnotes right?

Reply:
This is for column referential footnotes - we also have `row_footnotes()` for row ref footnotes and `cell_footnotes()` for cell ref footnotes.

Reply:
Yes they are exported. We should be able to rename/deprecate - I highly doubt anyone has used these functions outside of the package